cron/data/blob.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2021 Security Scorecard Authors
+//
+// Licensed under the Apache License, Vershandlern 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permisshandlerns and
+// limitathandlerns under the License.
+
+package data
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/ossf/scorecard/cron/config"
+	"gocloud.dev/blob"
+)
+
+const (
+	filePrefixFormat = "2006.01.02/150405/"
+)
+
+var (
+	errorWrite      = errors.New("blobWriter.Write")
+	errorWriteClose = errors.New("blobWriter.Close")
+	errorOpenBucket = errors.New("blob.OpenBucket")
+)
+
+func WriteToBlobStore(ctx context.Context, bucketURL, filename string, data []byte) error {
+	bucket, err := blob.OpenBucket(ctx, bucketURL)
+	if err != nil {
+		return fmt.Errorf("%w: %v", errorOpenBucket, err)
```

Comment:
nit: I don't think these fmt.Errorf calls are correct. See examples in https://blog.golang.org/go1.13-errors#TOC_3.3.

You should only have 1 error wrapped by %w, and this should be the underlying `err` that you got back from the previous call.


i.e.

`fmt.Errorf("blob.OpenBucket failed: %w", err)`